lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -218,18 +218,9 @@ DefParameterType('Expanded', sub {
     my $token    = $gullet->readXToken(0);
     my @tokens   = ();
     if ($token->getCatcode == CC_BEGIN) {
-      my $level = 1;
-      while ($token = $gullet->readXToken(0)) {
-        my $cc = $$token[1];
-        if ($cc == CC_END) {
-          $level--;
-          last unless $level; }
-        elsif ($cc == CC_BEGIN) {
-          $level++; }
-        push(@tokens, $token); }
-      return Tokens(@tokens); }
+      return scalar($gullet->readBalanced(1))->without_dont_expand; }
     else {
-      return $token; } },
+      return $token->without_dont_expand; } },
```

Comment:
I feel pretty good that this argument type should remove the noexpand flag conceptually. And I also feel somewhat good about suggesting that we erase `EExpanded` entirely and move all of those to the regular `Expanded`, as the uniform treatment should be able to handle both.

Reply:
Upon detailed review and reflection, this is indeed a great place for calling `->without_dont_expand`.